editor/src/components/canvas/canvas-strategies/strategies/draw-to-insert-text-strategy.tsx

```diff
@@ -107,24 +107,27 @@ export const drawToInsertTextMetaStrategy: MetaCanvasStrategy = (
         const isRoot = targetParentPathParts === 1
         const isClick = s === 'end-interaction' && interactionSession.interactionData.drag == null
         if (!isRoot && textEditableAndHasText && isClick) {
-          return strategyApplicationResult([
-            updateSelectedViews('on-complete', [targetParent.intendedParentPath]),
-            setCursorCommand(CSSCursor.Select),
-            wildcardPatch('on-complete', {
-              mode: {
-                $set: EditorModes.textEditMode(
-                  targetParent.intendedParentPath,
-                  canvasPointToWindowPoint(
-                    pointOnCanvas,
-                    canvasState.scale,
-                    canvasState.canvasOffset,
+          return strategyApplicationResult(
+            [
+              updateSelectedViews('on-complete', [targetParent.intendedParentPath]),
+              setCursorCommand(CSSCursor.Select),
+              wildcardPatch('on-complete', {
+                mode: {
+                  $set: EditorModes.textEditMode(
+                    targetParent.intendedParentPath,
+                    canvasPointToWindowPoint(
+                      pointOnCanvas,
+                      canvasState.scale,
+                      canvasState.canvasOffset,
+                    ),
+                    'existing',
+                    'no-text-selection',
                   ),
-                  'existing',
-                  'no-text-selection',
-                ),
-              },
-            }),
-          ])
+                },
+              }),
+            ],
+            'rerender-all-elements',
```

Comment:
definitely looks like one to fix

Reply:
Added comment about follow up in 38ec355d5.